build_runner/test/common/sdk.dart

```diff
@@ -5,13 +5,13 @@
 import 'dart:async';
 import 'dart:io';
 
-import 'package:cli_util/cli_util.dart';
 import 'package:path/path.dart' as p;
 import 'package:test/test.dart';
 import 'package:test_descriptor/test_descriptor.dart' as d;
 
-String _dartBinary = p.join(getSdkPath(), 'bin', 'dart');
-String _pubBinary = p.join(getSdkPath(), 'bin', 'pub');
+import 'package:build_runner/src/util/constants.dart';
+
+String _dartBinary = p.join(sdkBin, 'dart');
```

Comment:
```dart
final String dartPath =
    p.join(_sdkDir, 'bin', Platform.isWindows ? 'dart.exe' : 'dart');
```

Reply:
Don't for get the `.exe` for windows...

Reply:
We really should just put this in `pkg:io`

Reply:
Agreed that these things would be useful in a shared package. I don't think the `.exe` is required though.